examples/ceph.rs

```diff
@@ -17,35 +17,37 @@
 extern crate ceph;
 extern crate libc;
 
-use ceph::JsonData;
 #[cfg(target_os = "linux")]
 use ceph::admin_sockets::*;
 #[cfg(target_os = "linux")]
 use ceph::ceph as ceph_helpers;
 #[cfg(target_os = "linux")]
 use ceph::rados;
+use ceph::JsonData;
```

Comment:
Can we try to do things like rust-fmt separately; it massively increases the difficulty of reading through the changes on this review